style.css

```diff
@@ -75,18 +123,269 @@
+  .code-lang-container {
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+    padding-left: 1px;
+    padding-top: 0;
+    margin-top: 0;
+  }
+
+  .code-lang-container * {
+    border: solid #ebf0ee;
+    background-color: #ebf0ee;
+    border-radius: 8px;
+    color: #36b37e;
+    margin-right: 0.5em;
+    padding-right: 0.2em;
+    padding-left: 0.1em;
+  }
+
+  .ruby-on-rails {
+    flex: 5;
+  }
+
+  /* aside */
+  .aside-container {
+    max-width: 100%;
+    display: grid;
+    grid-template-columns: auto;
+    justify-items: center;
+    text-align: center;
+    background: url('images/illustration-about-me-mobile.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .header-container-about {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    min-width: 100%;
+    min-height: 30%;
+    background: url('images/mask-group.svg');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right;
+  }
+
+  #about-head {
+    font-size: 40px;
+    margin-top: 1.5em;
+  }
+
+  .about-paragraph {
+    margin-bottom: 20px;
+    margin-left: 17%;
+    width: 70%;
+    font-weight: lighter;
+  }
+
+  .btn {
+    width: 122px;
+    margin-left: 1em;
+  }
+
+  footer .btn {
+    width: 159px;
+  }
+
+  /* skillset */
+  .skillset {
+    max-width: 100%;
+    margin-top: 80px;
+    display: grid;
+    justify-items: center;
+    text-align: center;
+  }
+
+  .box-container {
+    border: 1px solid #c1c7d0;
+    background-color: #ebf0ee;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 1em;
+    width: 327px;
+    height: 352px;
+  }
+
+  .logo-container {
+    border-radius: 50%;
+    box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+    padding: 2%;
+    background-color: #fff;
+  }
+
+  .header-container {
+    margin-top: 4em;
+  }
+
+  .code-skill {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
     width: 80%;
+  }
+
+  .lang {
+    align-self: center;
+    font-size: small;
+    margin-left: 0;
+    margin-right: 5px;
+    margin-bottom: 10px;
+    padding-left: 2px;
+    padding-right: 2px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    border-radius: 8px;
+    background-color: #fff;
+    color: #36b37e;
+  }
+
+  #code-lang-list {
+    justify-content: center;
+    min-width: 100%;
+  }
+
+  #frame-wrk-list {
+    min-width: 70%;
+  }
+
+  .extra {
+    min-width: 100%;
+    padding: 0;
+  }
+
+  .last-head-container {
     text-align: center;
-    font-size: 20px;
+    display: flex;
+    justify-content: center;
+  }
+
+  .header-3 {
+    font-size: 40px;
     color: #172b4d;
+    margin-left: 0.5em;
+    margin-right: 0.5em;
   }
 
-  .nav-container > * {
-    margin-right: 1em;
+  .form {
+    display: flex;
+    flex-direction: column;
+    margin: 1em;
+  }
```

Comment:
You are supposed to leave a single line space at line 354 after your .nav-container styling.